Source/Locales/Main.lua

```diff
@@ -0,0 +1,28 @@
+ATR_L = {}
+
+local function fixMissingTranslations(incomplete, locale)
```

Comment:
Will we need this when we use the curse tooling?

Reply:
We probably won't

Reply:
But it will remain useful for offline testing